system-tests/CFortranInterop/expected.dump.txt

```diff
@@ -0,0 +1,33 @@
+File View
+  File   : system-tests/CFortranInterop/c_lower.c
+    Hash : 2457312520
+  File   : system-tests/CFortranInterop/c_roundtrip.c
+    Hash : 3255455825
+  File   : system-tests/CFortranInterop/f_inters.F90
+    Hash : 1710816464
+  File   : system-tests/CFortranInterop/f_inters.h
+    Hash : 3170915542
+  File   : system-tests/CFortranInterop/f_var.F90
+    Hash : 22378257
+  File   : system-tests/CFortranInterop/f_var.h
+    Hash : 58283018
+Fortran View
+  Program unit    : f_inters
+    Found in      : system-tests/CFortranInterop/working/f_inters.f90
+    Prerequisites : get_f_var_ptr
+  Program unit    : f_var
+    Found in      : system-tests/CFortranInterop/working/f_var.f90
+    Prerequisites : 
+C View
+  Symbol          : f_inter
```

Comment:
I think this is correct but the name "f_inter" is different to the program unit (and file) "f_inters" by only the trailing "s". This is a bit confusing. Can you think of a better name?

Reply:
Yes it could certainly make things clearer - although I think in the real world the names given are often close to each other (or even identical; but Fab can't handle that presently)